### PR TITLE
[IA-97] Add type definition and backend section for manage favourite language status bar

### PR DESCRIPTION
**Pull request by debiff**
This PR add the `favourite_language` section to the `Section` type definition and corresponding backend section for manage the favourite language status bar in the app.

### CODE REVIEW

File: status/backend.json [hunk 1]
```diff
@@ -169,6 +169,14 @@
         "it-IT": "",
         "en-EN": "."
       }
+    },
+    "favourite_language": {
+      "is_visible": false,
+      "level": "normal",
+      "message": {
+        "it-IT": "",
+        "en-EN": "."
```
Comment (Undermaken) [inline status/backend.json:178]:
```suggestion
        "en-EN": ""
```

**Review by Undermaken — APPROVED** [review]

File: status/backend.json [hunk 2]
```diff
@@ -169,6 +169,14 @@
         "it-IT": "",
         "en-EN": "."
       }
+    },
+    "favourite_language": {
+      "is_visible": false,
+      "level": "normal",
+      "message": {
+        "it-IT": "",
+        "en-EN": ""
```
Comment (thisisjp) [inline status/backend.json:178]: @debiff does this model support German as well?

Comment (Undermaken) [reply status/backend.json:178]: Nope. Remote locales are a separated world from app locales.
They will be support German too in the near future.
In this case the app will shown the message in italian (as a default fallback for the app)

Comment (thisisjp) [reply status/backend.json:178]: The idea here is to include a link to contribute improving the translation—ideally to the "create new issue" page on github

**Pull request merged**